interfaces/builtin/removable_media.go

```diff
@@ -42,6 +42,11 @@ const removableMediaConnectedPlugAppArmor = `
 # Mount points could be in /run/media/<user>/* or /media/<user>/*
 /{,run/}media/*/ r,
 /{,run/}media/*/** rwk,
+
+# Allow read-only access to /mnt to enumerate items.
+/mnt/ r,
+# Allow write access to anything under /mnt
+/mnt/*/** rwk,
```

Comment:
I actually recommended `/mnt/** rwk,` in the forum. While some applications adhere to /mnt/foo/bar, unlike /media with udisks, this is not a requirement for /mnt and the above would block toplevel accesses to /mnt when its used for its original purpose, which is for mounting a temporary filesystem (http://www.pathname.com/fhs/pub/fhs-2.3.html#PURPOSE13).